pipeline/deploy/cloudformation/morphology_pipeline.yml

```diff
@@ -775,6 +775,9 @@ Resources:
                             "Name": "WORKING_BUCKET",
                             "Value": "${WorkingBucket}" 
                           },
+                          {
+                            "Name": "REFERENCE_BUCKET",
+                            "Value": "${PersistentStackName}-reference"
```

Comment:
missing a `}`?

Reply:
good catch, it wasn't building but not giving me a useful error.